### ISSUE TITLE
Uploading nanpy firmware for Arduino

### ISSUE THREAD

**Issue by KyawZinLatt** [nanpy/nanpy-firmware#12]
I have problem with uploading firmware to Arduino. May be the way I try is wrong although I follow the instruction here https://github.com/nanpy/nanpy-firmware. Is there any other way to upload firmware to Arduino? I have tried the explanation here http://www.instructables.com/id/Arduino-Raspberry-Pi-Internet-Radio/?ALLSTEPS . It is really easy and working for me. But that version is version 0.7. Now I want to try for latest version. Can you show explain me the way to do it just simple way?

**Comment by astagi** [comment]
Hi @KyawZinLatt , did you try this http://nanpy.github.io/? After 0.8 make and upload are entirely made using Arduino IDE

**Comment by KyawZinLatt** [comment]
yeah I have tried that. To perform upload what else I need to do before uploading?

**Comment by astagi** [comment]
You need to call ./config.sh and then set the libraries you want to use in
cfg.h.

On Thu, Feb 26, 2015 at 4:41 PM, KyawZinLatt <notifications@github.com>
wrote:

> yeah I have tried that. To perform upload what else I need to do before
> uploading?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nanpy/nanpy-firmware/issues/12#issuecomment-76200084>.
>



-- 
Andrea Stagi (@4stagi) - Develover @Nephila
Job profile: http://linkedin.com/in/andreastagi
Website: http://4spills.blogspot.it/
Github: http://github.com/astagi

**Comment by KyawZinLatt** [comment]
This question may be quite easy for you but I dont know how to set the libraries to use in cfg.h. How to set that?

**Comment by astagi** [comment]
Just turn 0 into 1 if you want to use a particular library :) start with a
simple example like blink... which Arduino board are you using?

On Thu, Feb 26, 2015 at 7:12 PM, KyawZinLatt <notifications@github.com>
wrote:

> This question may be quite easy for you but I dont know how to set the
> libraries to use in cfg.h. How to set that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nanpy/nanpy-firmware/issues/12#issuecomment-76232418>.
>



-- 
Andrea Stagi (@4stagi) - Develover @Nephila
Job profile: http://linkedin.com/in/andreastagi
Website: http://4spills.blogspot.it/
Github: http://github.com/astagi

**Comment by KyawZinLatt** [comment]
Currently I am using Arduino Uno board
where to change that value?

**Comment by astagi** [comment]
You have to call ./configure.sh and then edit Nanpy/cfg.h. Here you can
find a compressed version of the firmware with some settings enabled:
https://dl.dropboxusercontent.com/u/35837126/nanpy-firmware.zip

Please read http://nanpy.github.io/ carefully

On Fri, Feb 27, 2015 at 3:58 PM, KyawZinLatt <notifications@github.com>
wrote:

> Currently I am using Arduino Uno board
> where to change that value?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nanpy/nanpy-firmware/issues/12#issuecomment-76406930>.
>



-- 
Andrea Stagi (@4stagi) - Develover @Nephila
Job profile: http://linkedin.com/in/andreastagi
Website: http://4spills.blogspot.it/
Github: http://github.com/astagi
